### DISCUSSION TITLE
Extended Church of the New Epoch Campaign

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#153] [Show and tell]
I knocked this up last night, an extended zealot campaign that adds in two more Bullfrog created levels with briefings to make a slightly longer campaign.

To use, paste this over your miss001.ini:

https://pastebin.com/HgAntchH

You will also need [C010L002.DAT](https://github.com/swfans/swars-levels/blob/main/LEVELS/C010L002.DAT) from the SWLevels repository (why haven't you got it already?).

This gives you two more missions, firstly the PlayStation version's unique early level 'Unrest' set in Phoenix that is unlocked after beating the first zealot mission. I had to shuffle some things around to make it fit, so you now have to beat this level to unlock the Santiago level Test Of Faith.

Secondly it gives you the completely unused second demo level set in Bangkok 'The Shining'. This a mid campaign level that I've set to come after the multi-level mission 8 (Tripoli, Buenos Aires, etc). This one I think is about right, but need more feedback on. Most of the enemies have miniguns, but the tougher ones have lasers, so this just about seems appropriate tech level. 

Both levels have banks to rob. The first is missing a netscan for it, not sure I should add one or not as you basically can't miss it (I wrote the netscans for the PS1 levels as they don't have such a thing in that version).

**Comment by geist22** [comment]
Great work! Finally got around to playing this, those are some interesting levels.

I find it a bit hard to belive the first one is supposed to be the second level in the Church campaign, the Minigung wielding Unguided group inside the ambush is quite dangerous, and bringing Uzis to a Minigun fight is... not very healthy. Though being before the 4th (03) mission feels more in place, even if the mission is quite short. Starting the level from the cmd gives your Acolytes all Miniguns, too.

Love missions with flying Church vehicles, they are so good and I really like their design. The agents in this level feel somewhat unresponsive, especially the group that lands after one building is destroyed. They didn't react to my Acolytes until I walked right between them, and then they either attacked or just started walking towards the bridge.

**Reply by Moburma** [reply]
> I find it a bit hard to belive the first one is supposed to be the second level in the Church campaign

Scout's honour! I agree it is a little bit tough if you're not careful, but this is kind of intended for expert players and it's a pretty short and slight level once you understand it, so I think it's ok. It really was the second level of the PS1 version, and it was the _literal_ second level there, you didn't have any choice of leaving it slightly later like you can do on PC if you found it hard. Sink or swim!

Anyway, I've updated it to include the latest improvements of the zealot campaign (weather etc):
https://pastebin.com/hKMZjUVs
